portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/search/searchportlet/SearchFacets.testcase

```diff
@@ -1090,6 +1090,89 @@ definition {
 		}
 	}
 
+	@description = "This is a use case for LPS-171290 - When configuring the Category Facet with the vocabulary layout the different ordering options don't take effect."
+	@priority = 3
+	test CanChangeSortOrderOfCategoriesUsingVocabularyLayout {
+		task ("Create Vocabulary and Categories") {
+			JSONCategory.addVocabulary(
+				groupName = "Guest",
+				title = "V");
+
+			var categoryId1 = JSONCategory.addCategory(
+				groupName = "Guest",
+				title = "Sales",
+				vocabularyName = "V");
+			var categoryId2 = JSONCategory.addCategory(
+				groupName = "Guest",
+				title = "Community",
+				vocabularyName = "V");
+			var categoryId3 = JSONCategory.addCategory(
+				groupName = "Guest",
+				title = "Product",
+				vocabularyName = "V");
+			var categoryId4 = JSONCategory.addCategory(
+				groupName = "Guest",
+				title = "Engineering",
+				vocabularyName = "V");
+		}
+
+		task ("Add Web Contents and categorize them") {
+			JSONWebcontent.addWebContent(
+				assetCategoryIds = "${categoryId1},${categoryId2},${categoryId3},${categoryId4}",
+				content = "WC Content",
+				groupName = "Guest",
+				title = "WC WebContent Title 1");
+
+			JSONWebcontent.addWebContent(
+				assetCategoryIds = "${categoryId1},${categoryId3},${categoryId4}",
+				content = "WC Content",
+				groupName = "Guest",
+				title = "WC WebContent Title 2");
+
+			JSONWebcontent.addWebContent(
+				assetCategoryIds = "${categoryId3},${categoryId4}",
+				content = "WC Content",
+				groupName = "Guest",
+				title = "WC WebContent Title 3");
+
+			JSONWebcontent.addWebContent(
+				assetCategoryIds = ${categoryId4},
+				content = "WC Content",
+				groupName = "Guest",
+				title = "WC WebContent Title 4");
+		}
+
+		task ("Search for the Web Contents and configure the widget to use the 'Vocabulary Layout'") {
+			SearchPortlets.searchEmbedded(searchTerm = "test");
+
+			SearchFacetPortlet.updateFacetConfiguration(
+				displayTemplate = "Vocabulary Layout",
+				portletName = "Category Facet");
+
+			Refresh();
+		}
+
+		task ("Assert the sort order is changed from 'Term Frequency Descending' to 'Term Frequency Ascending'") {
+			for (var categories : list "Engineering,Product,Sales,Community") {
+				AssertElementPresent(
+					categoryName = ${categories},
+					locator1 = "Search#CATEGORY_FACET_VOCABULARY_LAYOUT");
+			}
```

Comment:
Order matters here. This assertion does not care about order. Can we use `SearchFacetPortlet.viewFacetItemsSpecificOrder`?